### PR TITLE
Use MessageActionBar style declarations on pinned message card

### PR DESCRIPTION
**Pull request by luixxiul**
Fixes https://github.com/vector-im/element-web/issues/22444

|Before|After|
|---------|------|
|before|after|
||after|

This PR add `mx_MessageActionBar` style declarations to the non-interactive message action bar on pinned message card. This way it becomes possible to keep the style between those message action bars to be synced, by making the style declarations refer the variables defined for `mx_MessageActionBar`.

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

type: defect

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Use MessageActionBar style declarations on pinned message card ([\#8757](https://github.com/matrix-org/matrix-react-sdk/pull/8757)). Fixes vector-im/element-web#22444. Contributed by @luixxiul.

### CODE REVIEW

File: src/components/views/right_panel/PinnedMessagesCard.tsx [hunk 1]
```diff
@@ -154,16 +154,15 @@ const PinnedMessagesCard = ({ room, onClose }: IProps) => {
             />
         ));
     } else {
-        content = <div className="mx_PinnedMessagesCard_empty">
-            <div>
-                { /* XXX: We reuse the classes for simplicity, but deliberately not the components for non-interactivity. */ }
```
Comment (kerryarchibald) [inline src/components/views/right_panel/PinnedMessagesCard.tsx:159]: I think we should keep this comment to explain the non-interactive component is included by design.

File: src/components/views/right_panel/PinnedMessagesCard.tsx [hunk 2]
```diff
@@ -154,16 +154,15 @@ const PinnedMessagesCard = ({ room, onClose }: IProps) => {
...
+        content = <div className="mx_PinnedMessagesCard_empty_wrapper">
+            <div className="mx_PinnedMessagesCard_empty">
+                <div className="mx_MessageActionBar mx_PinnedMessagesCard_MessageActionBar">
                     <div className="mx_MessageActionBar_maskButton mx_MessageActionBar_reactButton" />
                     <div className="mx_MessageActionBar_maskButton mx_MessageActionBar_replyButton" />
                     <div className="mx_MessageActionBar_maskButton mx_MessageActionBar_optionsButton" />
                 </div>
 
-                <h2>{ _t("Nothing pinned, yet") }</h2>
+                <h2 className="mx_PinnedMessagesCard_empty_header">{ _t("Nothing pinned, yet") }</h2>
```
Comment (kerryarchibald) [inline src/components/views/right_panel/PinnedMessagesCard.tsx:165]: Can you use the `Heading` component here please?  https://github.com/matrix-org/matrix-react-sdk/blob/develop/src/components/views/typography/Heading.tsx

File: src/components/views/right_panel/PinnedMessagesCard.tsx (hunk 1 above)
Comment (luixxiul) [reply src/components/views/right_panel/PinnedMessagesCard.tsx:159]: I've misunderstood the comment, thanks

**Comment by luixxiul** [comment]
Addressed the comments.

I *really* like using `Header` components :-D

**Review by kerryarchibald — APPROVED** [review]
LGTM!

**Pull request merged**